nixos/modules/config/vte.nix

```diff
@@ -9,7 +7,7 @@ let
   vteInitSnippet = ''
     # Show current working directory in VTE terminals window title.
     # Supports both bash and zsh, requires interactive shell.
-    . ${pkgs.vte}/etc/profile.d/vte.sh
+    . ${pkgs.vte.override { gtkVersion = null; }}/etc/profile.d/vte.sh
```

Comment:
1. Shouldn't this be a `cfg.package` to begin with?
2. Shouldn't we get rid of `null`ities?

Reply:
[it doesn't work this  way I suppose](https://github.com/NixOS/nixpkgs/blob/5c5bdd7b938b0e055e980faa95f6fab1a2112fab/pkgs/development/libraries/vte/default.nix#L76)

Reply:
I forgot to cherry-pick a commit into the PR.



> Shouldn't this be a `cfg.package` to begin with?

I don't know, could be maybe.